.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/FileSystem"]
+	path = submodules/FileSystem
+	url = git@github.com:aspnet/FileSystem.git
```

Comment:
Thanks to @Praburaj 's follow-up. I just git cloned XRE via https and tried restore submodules. With git@github.com protocol, we do need to provide credentials. Using https urls for submodules is more convenient.